.github/workflows/main.yml

```diff
@@ -0,0 +1,50 @@
+name: CI
+
+on:
+  push:
+    branches:
+    - master
+  pull_request:
+
+jobs:
+  tests:
+    name: Python ${{ matrix.python-version }}
+    runs-on: ubuntu-latest
+
+    strategy:
+      max-parallel: 5
+      matrix:
+        python-version:
+          - "3.6"
+          - "3.7"
+          - "3.8"
+          - "3.9"
+
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+      - name: Upgrade packaging tools
+        run: python -m pip install --upgrade pip setuptools virtualenv
+      - name: Install dependencies
+        run: python -m pip install --upgrade tox
+      - name: Run tox targets for ${{ matrix.python-version }}
+        run: |
+          ENV_PREFIX=$(tr -C -d "0-9" <<< "${{ matrix.python-version }}")
+          TOXENV=$(tox --listenvs | grep "^py$ENV_PREFIX" | tr '\n' ',') python -m tox
+      - name: Generate coverage file
+        if: matrix.python-version == '3.9'
+        run: "TOXENV=coverage python -m tox"
+      - name: Report coverage
+        if: matrix.python-version == '3.9' && github.event_name == 'pull_request' && (github.event.action == 'opened' || github.event.action == 'reopened' || github.event.action == 'synchronize')
+        uses: 5monkeys/cobertura-action@master
+        with:
+          path: coverage.xml
+          repo_token: ${{ secrets.GITHUB_TOKEN }}
+          minimum_coverage: 85
+          show_line: true
+          show_branch: true
+          show_missing: true
+          link_missing_lines: true
+          link_missing_lines_source_dir: "src"
```

Comment:
We don't have a `src`-dir. Not sure what it should be

Reply:
Or perhaps it should be `djedi` or `djedi/src`